protocols/6f2d9b/6f2d9b.ot2.apiv2.py

```diff
@@ -0,0 +1,63 @@
+from opentrons import protocol_api
+
+metadata = {
+    'protocolName': 'COVID-19 Patient Sample Prep',
+    'author': 'Sakib <sakib.hossain@opentrons.com>',
+    'description': 'Custom Protocol Request',
+    'apiLevel': '2.9'
+}
+
+
+def run(ctx):
+
+    [p1000_mount, extraction_type, samples,
+        sample_vol] = get_values(  # noqa: F821
+        "p1000_mount", "extraction_type", "samples", "sample_vol")
+
+    if extraction_type == "single":
+        if not 1 <= samples <= 94:
+            raise Exception('''Please enter a sample number between 1
+                            and 94 for single extraction.''')
+    elif extraction_type == "double":
+        if not 1 <= samples <= 47:
+            raise Exception('''Please enter a sample number between
+                            1 and 47 for double extraction.''')
+
+    def pick_up(pip):
```

Comment:
I don't think this function is necessarily necessary for this application, but definitely fine to leave as is